site/assets/scss/_variables_project.scss

```diff
@@ -46,6 +48,16 @@ $td-sidebar-tree-root-color: #222 !default;
     background-color:black !important;
 }
 
+.showcase img {
+    margin: 0 30px;
+}
+
+#cover-title {
+    max-width: 800px;
```

Comment:
I think 500px will work for both small and larger screens.

Small screen - https://screenshot.googleplex.com/6QUiQJ96FwK7qwk

Larger screen - https://screenshot.googleplex.com/9udSepQgZfV2Jz4

Reply:
I liked the responsiveness as the width changes (so a bit wider on wider screens), but at this point, we're just down to personal preference I think.

Reply:
Thanks for sharing your thoughts! I value your input...

Would it be appropriate to consolidate the changes from both https://github.com/googleforgames/agones/pull/3382 and https://github.com/Kalaiselvi84/agones/pull/325 into https://github.com/googleforgames/agones/pull/3415? The latest PR contains the accurate go.mod and go.sum files.

Reply:
Yeah - if you want to take these changes, and whatever other changes you think were appropriate and build out a new PR, that works 👍🏻 and we can do a final review.